examples/tp_table.rs

```diff
@@ -0,0 +1,207 @@
+use anyhow::{anyhow, Result};
+use ascii_table::{Align, AsciiTable};
+use criterion::black_box;
+use lurk::{
+    eval::lang::{Coproc, Lang},
+    lem::{eval::evaluate, multiframe::MultiFrame, store::Store},
+    proof::{
+        nova::{public_params, NovaProver, PublicParams},
+        Prover,
+    },
+};
+use num_traits::ToPrimitive;
+use pasta_curves::pallas::Scalar as Fr;
+use statrs::statistics::Statistics;
+use std::{cell::OnceCell, sync::Arc, time::Instant};
+
+const PROGRAM: &str = "(letrec ((loop (lambda (K) (begin
```

Comment:
A nit, but one I feel strongly about: can we *please* not produce first-person example code that even vaguely suggest that case-sensitivity in variable names (or otherwise except where absolutely semantically required) is a suggested practice? It's really, a bad idea.